test/e2e/dev/main.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"k8s.io/klog/v2"
+	e2estorageutils "k8s.io/kubernetes/test/e2e/storage/utils"
+	windows "sigs.k8s.io/sig-storage-local-static-provisioner/test/e2e/windows"
+)
+
+func main() {
+	klog.InitFlags(nil)
+	defer klog.Flush()
+
+	flag.Parse()
+
+	klog.Infof("Remotely executing a program!")
+
+	node := &v1.Node{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "e2e-test-mauriciopoppe-windows-node-group-l64t",
+		},
+	}
+	hostExec := windows.NewHostExec()
```

Comment:
this program is only for windows case, should it in windows subdir?

Reply:
moved to `test/cmd/remote-exec-dev/`, `test/e2e/windows` is a library that has to be used from a main package